apps/api/e2e/utils/test-setup.ts

```diff
@@ -1,19 +1,23 @@
 /**
- * Test setup file that runs before each test suite
+ * Test setupFile, this is run before each test file https://vitest.dev/config/#setupfiles
+ *
+ * NOTE: The actual server under test is started inside a sub-process, hence in
+ *   memory things like caches, timeouts, and intervals cannot be interacted with.
+ *
+ * NOTE: that if you are running --isolate=false, this setup file will be run
+ *   in the same global scope multiple times. Meaning, that you are accessing the
+ *   same global object before each test, so make sure you are not doing the same
+ *   thing more than you need.
  *
- * This is used to set up global Jest configurations and hooks
  */
 import fs from "fs";
 import path from "path";
 import client from "prom-client";
 import { afterAll, afterEach, beforeAll, beforeEach, vi } from "vitest";
 
-import { ServiceRegistry } from "@/services/index.js";
-
 import { dbManager } from "./db-manager.js";
 
-const services = new ServiceRegistry();
-
+// TODO: is this log file needed if we use Pino?
 // Path to log file
 const logFile = path.resolve(__dirname, "../e2e-server.log");
```

Comment:
I don't totally understand why we have this log file, so I left it alone.  This PR is big enough, another can handle figuring this out.